src/main/webapp/create-event.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset = "UTF-8">
+    <title>STEP Capstone</title>
+    <link rel = "stylesheet" href = "style.css">
+    <script src = "script.js"></script>
```

Comment:
Remove spaces to match the style we agreed upon

Reply:
Will do!